### DISCUSSION TITLE
Seeing an error about the "off-static" folder in /public - can you help?

### DISCUSSION THREAD

**Discussion by angelsk** [statamic/cms#10613] [Q&A] [answered]
So I'm seeing an error in the logs which I believe is related to Statamic (I'm new to this, picking up an old codebase from previous devs).  But it also could be a Forge thing....?

I couldn't find anything via a Google search, so thought I'd ask here, if anyone has seen this before and what it's trying to do.

The folder is root created and I don't have access to change permissions (or to see contents), so I was hoping there was maybe a config setting I could change to switch the directory it was using for whatever purpose.... but I can't find a mention in the code at all.

Any help welcome!

The error is as follows (it's triggered 3-4 times every 5 minutes)

```
Error: warning: could not open directory 'public/off-static/': Permission denied  
[2024-08-12 09:55:02] production.ERROR: Process Class: Git
Command: 'git' 'status' '--porcelain'
```

And the relevant details:
PHP 8.2
Laravel Framework 10.48.20
Statamic 4.58.2

**Comment by jasonvarga** [comment, marked as answer]
`off-static` is not something Statamic would have intentionally created, so it must be specific to your project.

Do you have it listed in `config/statamic/git.php`?

**Reply by angelsk** [reply]
> `off-static` is not something Statamic would have intentionally created, so it must be specific to your project.
> 
> Do you have it listed in `config/statamic/git.php`?

It's not anywhere in the project. Good to know it's not a requirement.  Just trying to get root access to nuke it. Thanks!
